test/integration/AdminSpec.scala

```diff
@@ -36,7 +121,7 @@ class AdminSpec
     describe("adding a Part") {
 
       it("should work and redirect me to the part show page") {
-        val PartAddPage.PartConfig(partId, url, connectionPoolSize, commandKey, commandKeyGroup, proxy) = PartAddPage.createPart(ThreadPoolAddPage.createThreadPool)
+        val PartAddPage.PartConfig(partId, url, connectionPoolSize, commandKey, commandKeyGroup, Some(proxy)) = PartAddPage.createPart(ThreadPoolAddPage.createThreadPool._2)
```

Comment:
Perhaps split the tuple into its own line for clarity, as `._2` is not immediately obvious.

```scala
val (_, threadPoolConfig) = ThreadPoolAddPage.createThreadPool
val PartAddPage.PartConfig(partId, _, _, _, _, _) = PartAddPage.createPart(threadPoolConfig)
```

Likewise for L143.